Userland/Libraries/LibGUI/AbstractScrollableWidget.cpp

```diff
@@ -75,20 +75,26 @@ void AbstractScrollableWidget::mousewheel_event(MouseEvent& event)
 void AbstractScrollableWidget::custom_layout()
 {
     auto inner_rect = frame_inner_rect_for_size(size());
-    int height_wanted_by_horizontal_scrollbar = m_horizontal_scrollbar->is_visible() ? m_horizontal_scrollbar->min_height() : 0;
```

Comment:
Figure out a way to change LibGUI in one commit, and avoid any changes outside of LibGUI, then when it supports the new way, change Games / Services / misc (demos & devtools) in one commit for each section